src/e2e_test/perf_tool/pkg/deploy/steps/skaffold.go

```
@@ -85,6 +90,21 @@ func (s *skaffoldDeployImpl) Deploy(clusterCtx *cluster.Context) ([]string, erro
 	return []string{ns}, nil
 }
 
+// stderrSink returns the io.Writer to use for skaffold's stderr and a cleanup
+// func. When stderrFile is set, output is teed to both os.Stderr and the file
+// (opened in append mode so multiple skaffold invocations all land in the same
+// log).
+func (s *skaffoldDeployImpl) stderrSink() (io.Writer, func(), error) {
+	if s.stderrFile == "" {
+		return os.Stderr, func() {}, nil
+	}
+	f, err := os.OpenFile(s.stderrFile, os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0644)
```

golangci-lint (2.12.2)

[error] File is not properly formatted

(gofumpt)